client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider.tsx

```diff
@@ -42,9 +44,29 @@ function TierUpgradeSlider( {
 	const infoReferenceElement = useRef( null );
 	const componentClassNames = classNames( 'stats-tier-upgrade-slider', className );
 	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
-	const tiers = useAvailableUpgradeTiers( siteId );
+	const availableTiers = useAvailableUpgradeTiers( siteId );
 	const EXTENSION_THRESHOLD = 2; // in millions
 
+	// Filter out already purchased tiers.
```

Comment:
Hey @grzegorz-cp, Nice work!

Can I persuade you to put the code block in here, which is specifically left blank for the functionality:

https://github.com/Automattic/wp-calypso/blob/d672375d833f3845bc7fa4cf08a24501fff1a7ee/client/my-sites/stats/stats-purchase/use-available-upgrade-tiers.tsx#L64

Thanks 🙏

Reply:
Thanks @kangzj. I moved it but I had to keep the purchase hook in the original place.